tritonbench/operators/gdpa/math.py

```diff
@@ -59,8 +59,8 @@ def activation_string_to_int(s: str):
 
 @triton.jit
 def tanh(x):
-    # Tanh is just a scaled sigmoid
-    return 2 * tl.sigmoid(2 * x) - 1
+    exp_x2 = fast_expf(2.0 * x)
+    return fast_dividef((exp_x2 - 1), (1 + exp_x2))
```

Comment:
Isn't this numerically unstable? (Since you quickly get to `inf / inf`)

Reply:
I saw this PR: https://github.com/meta-pytorch/tritonbench/pull/506/files changes it to `2 * fast_dividef(1.0, 1.0 + fast_expf(-2.0 * x)) - 1.0`, I think my changes are little simple. Do we have the assumption the `x > 0`?

Reply:
I just realized this is changing the non-approximate path. (There is already a path to use fastmath instead.) I did a pass on the ALU ops (https://github.com/meta-pytorch/tritonbench/pull/595). I apologize for the lack of visibility; internally we have diff stacks but I'm not sure if this repo supports something like ghstack.